### ISSUE TITLE
[bitnami/zookeeper] /opt/bitnami/scripts/libzookeeper.sh: line 351: 3: value is required

### ISSUE THREAD

**Issue by pengmingming** [bitnami/charts#3289]
**Which chart**:
bitbnami/zookeeper zookeeper-5.21.2.tgz

**Describe the bug**
zookeeper 02:34:11.21 
zookeeper 02:34:11.21 Welcome to the Bitnami zookeeper container
zookeeper 02:34:11.21 Subscribe to project updates by watching https://github.com/bitnami/bitnami-docker-zookeeper
zookeeper 02:34:11.22 Submit issues and feature requests at https://github.com/bitnami/bitnami-docker-zookeeper/issues
zookeeper 02:34:11.22 
zookeeper 02:34:11.22 INFO  ==> ** Starting ZooKeeper setup **
zookeeper 02:34:11.24 WARN  ==> You have set the environment variable ALLOW_ANONYMOUS_LOGIN=yes. For safety reasons, do not use this flag in a production environment.
zookeeper 02:34:11.28 INFO  ==> Initializing ZooKeeper...
zookeeper 02:34:11.28 INFO  ==> No injected configuration file found, creating default config files...
zookeeper 02:34:11.37 INFO  ==> Adding server: zookeeper-0.zookeeper-headless.yunion.svc.cluster.local:2888:3888
zookeeper 02:34:11.38 INFO  ==> Adding server: zookeeper-1.zookeeper-headless.yunion.svc.cluster.local:2888:3888
zookeeper 02:34:11.38 INFO  ==> Adding server: zookeeper-2.zookeeper-headless.yunion.svc.cluster.local:2888:3888
/opt/bitnami/scripts/libzookeeper.sh: line 351: 3: value is required

**To Reproduce**
1. git clone https://github.com/bitnami/charts.git
2. cd  charts/bitnami/zookeeper
   // I'd like to overwrite the values.yaml use the values-production.yaml
3. cp values-production.yaml values.yaml
4. helm install zookeeper . -n yunion
5. kubectl get pods -n yunion 
    zookeeper-0               0/1     Error              4          103s
    zookeeper-1               0/1     Error              4          103s
    zookeeper-2               0/1     CrashLoopBackOff   4          102s
6. kubectl logs -f zookeeper-0  -n yunion

**Expected behavior**
zookeeper 02:34:11.21 
zookeeper 02:34:11.21 Welcome to the Bitnami zookeeper container
zookeeper 02:34:11.21 Subscribe to project updates by watching https://github.com/bitnami/bitnami-docker-zookeeper
zookeeper 02:34:11.22 Submit issues and feature requests at https://github.com/bitnami/bitnami-docker-zookeeper/issues
zookeeper 02:34:11.22 
zookeeper 02:34:11.22 INFO  ==> ** Starting ZooKeeper setup **
zookeeper 02:34:11.24 WARN  ==> You have set the environment variable ALLOW_ANONYMOUS_LOGIN=yes. For safety reasons, do not use this flag in a production environment.
zookeeper 02:34:11.28 INFO  ==> Initializing ZooKeeper...
zookeeper 02:34:11.28 INFO  ==> No injected configuration file found, creating default config files...
zookeeper 02:34:11.37 INFO  ==> Adding server: zookeeper-0.zookeeper-headless.yunion.svc.cluster.local:2888:3888
zookeeper 02:34:11.38 INFO  ==> Adding server: zookeeper-1.zookeeper-headless.yunion.svc.cluster.local:2888:3888
zookeeper 02:34:11.38 INFO  ==> Adding server: zookeeper-2.zookeeper-headless.yunion.svc.cluster.local:2888:3888
/opt/bitnami/scripts/libzookeeper.sh: line 351: 3: value is required

**Version of Helm and Kubernetes**:

- Output of `helm version`:

```
version.BuildInfo{Version:"v3.2.4", GitCommit:"0ad800ef43d3b826f31a5ad8dfbb4fe05d143688", GitTreeState:"clean", GoVersion:"go1.13.12"}
```

- Output of `kubectl version`:

```
Client Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.5", GitCommit:"e6503f8d8f769ace2f338794c914a96fc335df0f", GitTreeState:"clean", BuildDate:"2020-06-26T03:47:41Z", GoVersion:"go1.13.9", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"17", GitVersion:"v1.17.6", GitCommit:"d32e40e20d167e103faf894261614c5b45c44198", GitTreeState:"clean", BuildDate:"2020-05-20T13:08:34Z", GoVersion:"go1.13.9", Compiler:"gc", Platform:"linux/amd64"}
```

**Additional context**

**Comment by pengmingming** [comment]
Hi @joancafom any ideas?

**Comment by marcosbc** [comment]
Hi @pengmingming, thanks for reporting this. We have been able to reproduce it and are working on a fix. We'll keep you posted.

**Comment by pengmingming** [comment]
@marcosbc Alright, thanks for your quick replay.

**Comment by marcosbc** [comment]
Hi @pengmingming, this issue was fixed with the tag `3.6.1-debian-10-r88` of the Bitnami Docker image for ZooKeeper. If you upgrade your chart to 5.21.3 you should see the affected change.
